open-apparel-registry/open-apparel-registry
# Replace technology vendor link & favicon

## Overview

- replace technology vendor link in the footer
- replace favicon.ico with an empty file: previously it was the prior vendor's logo, but we'll eventually want to replace it with something OAR specific

Connects #90

## Demo

<img width="376" alt="screen shot 2019-01-29 at 2 38 37 pm" src="https://user-images.githubusercontent.com/4165523/51935296-96684c00-23d3-11e9-833e-3deee04381bf.png">


## Testing Instructions

- serve this branch and visit :6543
- verify that you see the new link in the footer and that displays as expected
- verify that there's no favicon displayed
